src/AutoRest.CSharp/Mgmt/AutoRest/MgmtOutputLibrary.cs

```diff
@@ -196,6 +198,20 @@ public override CSharpType FindTypeForSchema(Schema schema)
             return provider?.Type;
         }
 
+        public LongRunningOperationInfo FindLongRunningOperationInfo(OperationGroup operationGroup, Operation operation)
+        {
+            var mgmtRestClient = FindRestClient(operationGroup);
+
+            var nextOperationMethod = operation.Language.Default.Paging != null
+                ? mgmtRestClient.GetNextOperationMethod(operation.Requests.Single())
+                : null;
+
```

Comment:
Feel like we need an assert over the client not being null here.  Also, null check on operation parameter before deref.

Reply:
And I think ```operation?.Language?.Default?.Paging``` is safest here.